package.json

```diff
@@ -134,7 +134,11 @@
 		"fixtures:generate": "npm run fixtures:server-registered && cross-env GENERATE_MISSING_FIXTURES=y npm run test-unit",
 		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
 		"package-plugin": "./bin/build-plugin-zip.sh",
+		"postinstall": "lerna bootstrap --hoist",
 		"pot-to-php": "./bin/pot-to-php.js",
+		"publish:check": "lerna updated",
+		"publish:dev": "lerna publish --npm-tag next",
+		"publish:prod": "lerna publish",
```

Comment:
Adding docs to the RELEASE.md would be good (or contributing) to explain how to use these things :), could be done later once the config is more battle-tested.

Reply:
Copied https://github.com/WordPress/gutenberg/blob/37a722322327ee45f0510e0c7fa9a2ec2f793db2/CONTRIBUTING.md#releasing-packages from the existing docs in `WordPress/packages` :)
Props to @youknowriad and @pento 🙇